app/serializers/optimized_project_serializer.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+# Note: This is _not_ an active model serializer. To use, instantiate and
+# call #serialize
+class OptimizedProjectSerializer
```

Comment:
Is it worth looking at using jb in libraries also?

Reply:
Totally worth exploring.